TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AutonomousRight.java

```diff
@@ -611,66 +611,64 @@ private void autonomousCore() {
             // push sleeve cone out, and reading background color for calibration
-            robotRunToPosition(36.0, true);
+            robotRunToPosition(39.0, true);
         }
         parkingLocation = calculateParkingLocation(sleeveColor, backgroundColor);
         Logging.log("Autonomous - parking lot aisle location: %.2f", parkingLocation);
 
         // lift slider during strafe to high junction
         setSliderPosition(HIGH_JUNCTION_POS);
-        robotRunToPosition(-4.0, true); // get rid of sleeve cone
+        robotRunToPosition(-7.0, true); // get rid of sleeve cone
 
 
-        robotRunToPosition(-12.0, false); // strafe robot half mat to left side
-        waitSliderRun(); // make sure slider has been lifted.
+       // turn robot 45 degrees left
+        Orientation imuAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
+        rotate(45 - AngleUnit.DEGREES.normalize(imuAngles.firstAngle), AUTO_ROTATE_POWER);
 
-        robotRunToPosition(7.5, true); // drive robot V to high junction
+        //drive forward & drop
+        robotRunToPosition(13.1, true);
         autoUnloadCone();
-        Logging.log("Autonomous - pre-load cone has been unloaded.");
-        // lower down slider after unloading cone
-        setSliderPosition(WALL_POSITION);
-
-        Orientation imuAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
+        robotRunToPosition(13.1, true);
 
         for(int autoLoop = 0; autoLoop < 2; autoLoop++) {
             Logging.log("Autonomous - loop index: %d ", autoLoop);
 
-            // right turn 90 degree
+            // right turn 135 degree
             imuAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
             Logging.log("Autonomous - imu angle before right turn: %.2f", imuAngles.firstAngle);
-            rotate(-AngleUnit.DEGREES.normalize(imuAngles.firstAngle) - 90, AUTO_ROTATE_POWER);
+            rotate(-AngleUnit.DEGREES.normalize(imuAngles.firstAngle) - 135, AUTO_ROTATE_POWER);
             Logging.log("Autonomous - imu turn: %.2f degree", -AngleUnit.DEGREES.normalize(imuAngles.firstAngle) - 90);
             Logging.log("Autonomous - imu angle after turn: %.2f", lastAngles.firstAngle);
 
-            // strafe to the left a little bit to compensate for the shift from 90 degree rotation(currently 1 inch).
-            robotRunToPosition(-1.0, false);
+            // strafe to the left a little bit to compensate for the shift from 135 degree rotation(currently 1.5 inch).
+            robotRunToPosition(-1.5, false);
 
-            // adjust position and double rotation for accurate 90
+            // adjust position and double rotation for accurate 135
             imuAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
-            rotate(-AngleUnit.DEGREES.normalize(imuAngles.firstAngle) - 90, AUTO_ROTATE_POWER);
```

Comment:
here should be "-AngleUnit.DEGREES.normalize(imuAngles.firstAngle) - 90", because we want the robot to turn -90 degrees compared to the initial orientation.